### DISCUSSION TITLE
White bar in File explorer

### DISCUSSION THREAD

**Discussion by zayhden** [Open-Shell/Open-Shell-Menu#1113] [General]
When using the dark theme on 

windows and downloading Open shell there is a white bar that replaces the black one
![White bar in file explorer](https://user-images.githubusercontent.com/107539775/185799922-be6d20c0-8bb9-4552-affa-519968f0e792.png)

**Comment by blackcrack** [comment]
Windows Version, 
Open-Shell Version

?

**Reply by zayhden** [reply]
Windows 10, and using the pre release version 

note: this has happened when i used the normal version

**Reply by blackcrack** [reply]
which pre-release 4.4.170 or the 4.4.174 ?

**Reply by zayhden** [reply]
.174 

i just found some settings im able to change in the file explorer itself that have to do with open shell. It has to do with the statusbar not changing color. There doesn't seem to be an option for changing its color.
![image](https://user-images.githubusercontent.com/107539775/185800419-1b8e02ff-82d1-4e09-83a2-701e1d0a06d0.png)

**Reply by zayhden** [reply]
I've had this issue when using other versions too

**Comment by zayhden** [comment]
Update* 
 
I have found a solution.

**Reply by YesTrustMe** [reply]
Lol. You found a solution, That's nice. Care to share? #sophism

**Reply by zayhden** [reply]
Sure thing! So what worked for me is right-clicking on the openshell classic explorer bar to access the options and un-selecting 'show status bar'. If that doesnt work you should go to the file explorer options and re-selecting 'show status bar' after trying it the first way I explained above here. 

Feel free to ask questions. Im not sure if i'll be able to answer them all since it took me trial and error to get this to work, but i can give it a go.
